airflow/task/task_runner/base_task_runner.py

```diff
@@ -87,11 +86,7 @@ def __init__(self, local_task_job):
 
         self._error_file = NamedTemporaryFile(delete=True)
         if self.run_as_user:
-            try:
-                os.chown(self._error_file.name, getpwnam(self.run_as_user).pw_uid, -1)
-            except KeyError:
-                # No user `run_as_user` found
-                pass
+            subprocess.call(['sudo', 'chown', self.run_as_user, self._error_file.name], close_fds=True)
```

Comment:
```suggestion
            subprocess.check_call(['sudo', 'chown', self.run_as_user, self._error_file.name], close_fds=True)
```

I think

Reply:
I agree that you're probably right it should be `check_call`. I simply stole the the subprocess call that runs a chown a few statements above so maybe they should both change?

Reply:
Yeah -- could you change them both?